npm/util/osutil_linux/current-azure-chains.go

```diff
@@ -0,0 +1,62 @@
+//go:build !windows
+// +build !windows
+
+package linuxutil
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/Azure/azure-container-networking/npm/util"
+	npmerrors "github.com/Azure/azure-container-networking/npm/util/errors"
+	"k8s.io/klog"
+	utilexec "k8s.io/utils/exec"
+)
+
+const (
+	azureChainGrepPattern   string = "Chain AZURE-NPM"
+	minAzureChainNameLength int    = len("AZURE-NPM")
```

Comment:
move `AZURE-NAME` into a const of it's own. You can then use it in `azureChainGrepPattern`. If we ever have to change the name in the future (there are discussions on renaming things IIRC) we can change it one place.

Reply:
using a constant now